hack/jenkins/e2e.sh

```diff
@@ -649,10 +649,10 @@ case ${JOB_NAME} in
     : ${E2E_DOWN:="true"}
     : ${E2E_TEST:="false"}
     : ${USE_KUBEMARK:="true"}
-    : ${NUM_MINIONS:="40"}
-    : ${E2E_ZONE:="asia-east1-a"}
-    : ${MASTER_SIZE:="n1-standard-32"}  # Note: not available in all zones
-    : ${MINION_SIZE="n1-standard-16"}   # Note: can fit about 17 hollow nodes per core
+    : ${NUM_MINIONS:="8"}
+    E2E_ZONE="asia-east1-a"
```

Comment:
can you please move this line to the end (similarly to what we do in scalability)?